ons_alpha/jinja2/templates/pages/bulletin_page.html

```diff
@@ -107,11 +107,16 @@ <h1 class="ons-u-fs-3xl bulletin-header__heading">
             {% for block in page.body %}
                 {% if block.block_type == "heading" %}
                     {% if ns.has_section %} </section>{% endif %}
-                    <section id="{{ block.value|slugify }}">
+                    <section
+                        id="{{ block.value|slugify }}"
+                        class="streamfield {% if last_flush %}streamfield--last-flush{% endif %}"
+                    >
                         <h2>{{ block.value }}</h2>
                         {% set ns.has_section = True %}
                 {% else %}
-                    {% include_block block %}
+                    <div class="streamfield {% if loop.last %}streamfield--last-flush{% endif %}">
+                        {% include_block block %}
```

Comment:
doesn't this also come with a `<div class="streamfield">` wrapper?

Reply:
ah, it does not, but potentially has a different problem (if you have several SF blocks within the section)

or am I getting this wrong? sorry, Friday brain!

<img width="1253" alt="Screenshot 2025-02-14 at 11 16 55" src="https://github.com/user-attachments/assets/e4d7ea69-b496-4ddd-a29f-ab6633f9ef60" />

Reply:
Basically the div adds the extra spacing when there are several streamfield blocks within one section. It is possible that you get a nested div and section, both with a margin bottom, but in that situation the end result is still a margin bottom of 4 rem because it collapses.

There could still be an issue if an editor, say, added a heading above a document block - but they should not do that because the document block has it's own heading 2.

If we think it is a bit risky I would be happy to just revert the change to the section markup and not worry that the section highlighting doesn't quite work on the bulletin page - it feels more important for the spacing to be correct.

Reply:
understood. let's deploy this to staging and maybe get input from the editors? (or do they not have access to staging).

it would be safer to deploy the minimal changes, and follow this up next week (say on Tue?)